README.md

```diff
@@ -1,2 +1,51 @@
 # test
 
+## Overview 
+
+This is the home to automated end-to-end tests for the Crossplane Infrastructure. It contains logic and configuration files for the testing of Crossplane and its stack. 
+
+It uses Go and Github actions primarily to run tests and automated workflows. To create a new workflow place a workflow file in the `.github` directory of the repository. These workflows install crossplane into a `kind` cluster to run different test scenarios. 
```

Comment:
```suggestion
It uses Go and Github actions primarily to run tests and automated workflows. To create a new workflow place a workflow file in the `.github` directory of the repository.
```

I think we can omit last line since they are not strictly required to execute in that manner.